### PR TITLE
New Cargo Missions (Pirate Bounties, Salvage)

### PR DESCRIPTION
**Pull request by generalthrax**
## About The Pull Request

A few more missions to cargo. Should make them more varied, with some financial incentive for combat ships or scrappers to search for certain things

- [x] Add dogtags to Frontiersmen corpses
- [x] Add dogtags to Ramzi drops or have them spawn corpses with dogtag on them (ramzi mobs are kind of stupid i might change them)
- [x] Frontie Mission
- [x] Ramzi Mission
- [x] Turn in Protolathe Mission
- [x] Turn in RND Console Mission (may combine all three of these into one)
- [x] Remove Fishing Missions
- [x] Remove impossible missions like Capture Abandoned Floorbot and Capture Bluespace Watcher
- [x] Descriptions
- [x] Increase value of migo mission
- [x] Increase value of Strange Crystal mission
- [x] Dogtag sprites (courtesy of @rye-rice 

(prices subject to change)

## Why It's Good For The Game

More financial incentives to go hunt down the Pirate ruins for combat ships. A reason to stash up the rare machinery you find from ruins to sell them instead of Grinding RND And Getting Those Sweet Research Points

Crew bonding session of stripping dogtags from pirates, ensuring their bodies never return to their loved ones and you can rack in that sweet, sweet Inteq cash

## Changelog

:cl:
add: Adds Dogtag Missions to hunt down Ramzi Clique and Frontiersman NPCS from Ruins
add: Adds Salvage Missions to collect Protolathe or R&D Console Boards
del: Removes Fishing Missions
del: Removes impossible missions like Bluespace Watcher and Abandoned Floorbot
balance: Increases price of Migo missions to be on par with Legion
balance: Increases the value of Strange Crystal missions
del: Remove R&D design board from unlocked Protolathes and Science Protolathes
/:cl:

### CODE REVIEW

**Comment by Martinpachu** [comment]
who would EVER sell their RND? other than me

**Comment by generalthrax** [comment]
Plenty would. Its also weighed to be Quite Rare so you're not stocking up like 8 RND boards for a shiptillion credits

**Comment by Madtankdriver** [comment]
The dog tags on corpses sounds like a pretty cool idea. I like it!

**Comment by Martinpachu** [comment]
> Plenty would. Its also weighed to be Quite Rare so you're not stocking up like 8 RND boards for a shiptillion credits

its less that you can get a bunch of them and more so that it's so rare and absurdly powerful that nobody would ever do it. i like it, though.

**Comment by generalthrax** [comment]
I think you underestimate just how many people dislike interacting with RND. Outside of Gamers, at least. It'll be very low-weighted anyways, so w/e.

**Comment by Martinpachu** [comment]
> I think you underestimate just how many people dislike interacting with RND. Outside of Gamers, at least. It'll be very low-weighted anyways, so w/e.

i dont think the weighing matters a lot since rnd is sooo rare either way. not really a problem i just thought it was a little funny.

**Comment by Aquidu** [comment]
Thoughts on a potential mission where you have to turn in a canister of a certain gas?

**Comment by Apogee-dev** [comment]
> > I think you underestimate just how many people dislike interacting with RND. Outside of Gamers, at least. It'll be very low-weighted anyways, so w/e.
> 
> i dont think the weighing matters a lot since rnd is sooo rare either way. not really a problem i just thought it was a little funny.

Don't worry! it's going to get rarer.

**Comment by thgvr** [comment]
...are ramzi/frontiersmen mobs common enough for this?

**Comment by generalthrax** [comment]
> ...are ramzi/frontiersmen mobs common enough for this?

Yes.

**Comment by generalthrax** [comment]
This should be ready. I'm aware about Fallcon's dynamic mission stuff but I figured I'd get this out of the way to see if its compatible rather than just close this outright

**Review by MarkSuckerberg — APPROVED** [review]
still seems good for merge if you can manage to get a sprite for the dogtags and resolve the conflict since I think dynamic missions are still a little ways off

**Comment by generalthrax** [comment]
Sounds good. I'll try and get it done once this uptime is over.

**Comment by generalthrax** [comment]
Conflicts are fixed and I now have a sprite courtesy of @rye-rice. This should be good to go.

**Comment by Sun-Soaked** [comment]
fix checks and conflicts and I'll get it in

File: code/modules/research/designs/comp_board_designs.dm [hunk 1]
```diff
@@ -182,14 +182,6 @@
...
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_CARGO
 
-/datum/design/board/rdconsole
-	name = "Computer Design (R&D Console)"
-	desc = "Allows for the construction of circuit boards used to build a new R&D console."
-	id = "rdconsole"
-	build_path = /obj/item/circuitboard/computer/rdconsole
-	category = list("Computer Boards")
-	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
-
```
Comment (MarkSuckerberg) [inline code/modules/research/designs/comp_board_designs.dm:192]: you gotta remove this from the base research node to get it to pass checks. or just readd this

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]
also you gotta add a worn state for the dogtags too

**Review by MarkSuckerberg — APPROVED** [review]

**Pull request merged**